### DISCUSSION TITLE
Docker & Docker Compose Script Issue

### DISCUSSION THREAD

**Discussion by AKHwyJunkie** [tteck/Proxmox#180] [General]
First, thanks for all your hard work in putting these scripts together.  I played with a few awhile back.  The v3 scripts make them suitable for hassle-free production and was exactly what I wanted.

I'm having a bit of trouble with Docker Compose in the Docker image.  I'm guessing something is different from a normal install of Docker Compose?  For example:

```
root@docker:/# docker-compose -version
-bash: docker-compose: command not found
```
I could probably tear apart the install and bashrc the executable.  The real question, though, is whether this was intended?  FWIW, I noticed this with either Portainer installed or not.

Again, thanks for your contributions.  Your HAOS script saved me a bunch of time and hassle on my most recent rebuild.

**Comment by tteck** [comment]
You're used to docker compose v1 try `docker compose version`
```
root@docker:~# docker compose version
Docker Compose version v2.4.1
```

**Comment by AKHwyJunkie** [comment]
Sure enough, I'm the dummy.  I should have tried that as I was aware of the difference.  Oh well, thanks for setting me straight.
